snet_cli/commands.py

```diff
@@ -435,6 +429,7 @@ def call(self):
 
 
 class RegistryCommand(BlockchainCommand):
+    # Warning: none of these commands with the new Registry
```

Comment:
none of these commands *work* with the new Registry

Reply:
Fixed